frontend/src/components/widgets/DataGrid/DataGrid.tsx

```diff
@@ -50,11 +51,40 @@ function DataGrid({
     // This is a styled-components theme provider (not emotion!).
     // It is required by glide-data-grid to customize the theming.
     <ThemeProvider
-      theme={
-        {
-          /* TODO(lukasmasuch): Theme will be added in a later PR */
-        }
-      }
+      theme={{
```

Comment:
nit: I think this would be a bit cleaner defined as a const in the same scope as the `useTheme` call rather than in the `tsx`